enterprise/server/test/integration/ci_runner/BUILD

```diff
@@ -4,12 +4,6 @@ go_test(
     name = "go_default_test",
     srcs = ["ci_runner_test.go"],
     data = ["//enterprise/server/cmd/ci_runner"],
-    # TODO: Make this automatic when we figure out a good way to include
-    # a `git` binary in this test (since the action runner needs it).
-    # Also would need to support a BAZEL_BINARY env variable or flag
-    # to tell it to use a fixed version of Bazel that we include via
-    # data dep.
```

Comment:
is this no longer an issue?

Reply:
Nope, the default execution image includes both git and bazelisk, so the test works as-is. I tried running this test with remote execution locally, and it works (also it works with prod RBE, since the PR checks pass)

Reply:
It's a little bit of a bummer that this is not really hermetic though -- it means if someone checks out our code and runs all the tests without having git installed, this will fail. Right?

Reply:
True -- removed the implicit deps on git and bazelisk (using go-git instead of git, and using bazel from `//server/testutil/bazel`